movies-app/src/components/Genre/Genre.test.js

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
+import user from '@testing-library/user-event';
+import Genre from './Genre';
+
+describe('Genere', () => {
+
```

Comment:
Please remove blank lines if not required, also apply proper indentation throughout the app

Reply:
@shubhamagarwal  removed extra spaces